DueTimer.h

```diff
@@ -102,8 +102,8 @@ extern DueTimer Timer6;
 extern DueTimer Timer7;
 extern DueTimer Timer8;
 
+// _SAM3XA_
 #endif
 
-#else
-	#error Oops! Trying to include DueTimer on another device?
```

Comment:
Let's keep that Error! It saves people sometimes when forgetting to change the board...